src/commands/autocomplete/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import Command from 'cli-engine-command'
+import path from 'path'
+import {CustomColors} from 'cli-engine-command/lib/output'
+
+export class AutocompleteBase extends Command {
+  get autocompletePath (): string {
+    return path.join(this.config.dataDir, 'client', 'node_modules', 'cli-engine', 'autocomplete')
+  }
+
+  errorIfWindows () {
+    if (this.config.windows) {
+      this.out.error('Autocomplete is not currently supported in Windows')
+    }
+  }
+}
+
+export default class Autocomplete extends AutocompleteBase {
+  static topic = 'autocomplete'
+  static description = 'display autocomplete instructions'
+  // hide until beta release
+  static hidden = true
+  static args = [{name: 'shell', description: 'shell type', required: false}]
+
+  async run () {
+    this.errorIfWindows()
+
+    const shell = this.argv[0] || this.config.shell
+    if (!shell) {
+      this.out.error('Error: Missing required argument shell')
+    }
+
+    switch (shell) {
+      case 'bash':
+        this.out.log('Bash autocomplete coming soon')
+        // this.out.log('Symlink the autocomplete function via:')
+        // this.out.log()
+        // let fnFile = path.join(this.autocompletePath, 'bash', 'heroku')
+        // this.out.log(CustomColors.cmd(`$ ln -s ${fnFile} /usr/local/etc/bash_completion.d/heroku`))
+        break
```

Comment:
@RasPhilCo I would return here since you do not want to print 'Lastly, restart your shell' for bash